### DISCUSSION TITLE
get measurement

### DISCUSSION THREAD

**Discussion by YTheaudin96t** [geoman-io/leaflet-geoman#1346] [Questions & Help]
Hello,
I want to write the measurements in a textbox or retrieve the measurements of an item
Do you have a solution?

Best regards,
Yvan

**Comment by Falke-Design** [comment]
Hi @YTheaudin96t,
thank you for your question. Measurement is one of our [Geoman-Pro](https://geoman.io/leaflet-geoman-pro) features.

**Reply by YTheaudin96t** [reply]
Thanks I have the pro version, do you have any solution

**Reply by YTheaudin96t** [reply]
Je ne suis pas dans le bon sujet pour la version pro ?

**Reply by Falke-Design** [reply]
Everything is fine, I just didn't know that you have the pro version.

You can access the measurement data over a property on the layer (if the measurement was already calculated once): `layer.pm.measurements`.

**Reply by YTheaudin96t** [reply]
Thanks, :)
